hudi-cli/src/main/resources/application.yml

```diff
@@ -20,4 +20,7 @@ spring:
   shell:
     history:
       enabled: true
-      name: hoodie-cmd.log
\ No newline at end of file
+      name: hoodie-cmd.log
+    command:
+      version:
+        template: "classpath:version.txt"
```

Comment:
Not very familiar with the Spring stuff, are we fixing the spring cmd?

Reply:
The version command in CLI is not showing the current Hudi version.

Reply:
It was working until this file got removed from the hudi released. https://github.com/apache/hudi/blob/release-0.11.0/hudi-cli/src/main/java/org/apache/hudi/cli/HoodieSplashScreen.java#L60